src/models/Factor.js

```diff
@@ -161,10 +171,10 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
           return (provider === 'OKTA' && _.contains(['push', 'call'], factorType));
```

Comment:
I think nanda is the best person to ask about this. @nandagopalseshagiri-okta I assumed this would be like sms and would call transaction.verify on resend. Will we have the ability to call transaction.resend instead?